src/Transmogrification.cpp

```diff
@@ -792,7 +792,7 @@ bool Transmogrification::SuitableForTransmogrification(Player* player, ItemTempl
             return false;
     }
 
-    if (!IgnoreReqLevel && player->GetLevel() < proto->RequiredLevel)
+    if (!IgnoreReqLevel && IsPlusFeatureEligible(player->GetGUID(), PLUS_FEATURE_SKIP_LEVEL_REQ) && player->GetLevel() < proto->RequiredLevel)
```

Comment:
Shouldn't these be `!IsPlusFeatureEligible(....)`? With the way it currently is, level reqs will be globally ignored by default because `&& false` in this check, leading to the return false here never being called